WordPressKit/AccountServiceRemoteREST+SocialService.swift

```diff
@@ -37,9 +35,8 @@ extension AccountServiceRemoteREST {
         ] as [String: AnyObject]
         
         // Append Apple specific parameters
-        if service == .apple {
-            params["user_email"] = appleEmail as AnyObject
-            params["user_name"] = appleFullName as AnyObject
+        if service == .apple, let appleConnectParameters = appleConnectParameters {
```

Comment:
We could just always append the parameters regardless of the service, then we don't need special conditional code for Apple?

Reply:
I figured it's better not to append them just in case, but ok.